terraform/environments/analytical-platform-compute/iam-policies.tf

```
@@ -280,3 +280,58 @@ module "analytical_platform_lake_formation_share_policy" {
 
   policy = data.aws_iam_policy_document.analytical_platform_share_policy.json
 }
+
+data "aws_iam_policy_document" "apc_bucket_logs_kms_key_policy" {
```

rename to accurately reflect policy resources
```suggestion
data "aws_iam_policy_document" "s3_access_logs_kms_key_policy" {
```